lib/safira_web/controllers/attendee_controller.ex

```diff
@@ -23,6 +24,9 @@ defmodule SafiraWeb.AttendeeController do
         render(conn, "manager_show.json", attendee: attendee)
 
       true ->
+        attendee = 
+          attendee
+          |> Map.put(:redeemables,Store.get_attendee_redeemables(attendee))
```

Comment:
This attendee comes from line `get_attendee!` in line 17. If you noticed this function already preloads the association with redeemables. Then in function `get_attendee_redeembales`, you're repeating this preload. You should decide where to preload. But remember that you use this function in other places, so I think you should preload the redeemables inside `get_attendee_redeemables` and not inside `get_attendee!`, but the decision is yours.